src/test/scala/com/cognite/sdk/scala/v1/AssetsTest.scala

```diff
@@ -64,6 +64,25 @@ class AssetsTest extends SdkTest with ReadBehaviours with WritableBehaviors {
     }
   )
 
+  it should "support updating by externalId" in {
+     val createAssets = Seq(
+      Asset(name = "update-1", externalId = Some("update-1-externalId")),
+       Asset(name = "update-2", externalId = Some("update-2-externalId")))
+    client.assets.createFromRead(createAssets)
+    import com.cognite.sdk.scala.common.Set
```

Comment:
We should change this to `SetTo` or similar instead so that we avoid the name collision with the scala `Set`